.github/workflows/test-languages.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: Test Languages
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: Test Languages
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout PLCC
+        uses: actions/checkout@v3
+        with:
+          repository: ourPLCC/plcc
+      -
+        name: Build test image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./containers/plcc/Dockerfile
+          tags: test-image:latest
+          push: false
+      -
+        name: Test Languages
+        uses: addnab/docker-run-action@v3
+        with:
+          image: test-image:latest
+          run: |
+            git clone https://github.com/ourPLCC/languages.git
```

Comment:
This will always test against the main branch, even when pushing code to a feature branch.

The current working directory should contain the languages code from the feature branch. So we should mount it into the container when we run it. Try something like this...

```yaml
      -
        name: Test Languages
        uses: addnab/docker-run-action@v3
        with:
          image: test-image:latest
          options: -v ${{ github.workspace }}:/work
          run: |
            /work/bin/test.bash
```

Reply:
If you were to use that code without first switching into languages, the plcc repo and its directories appear, for example the installers, src, tests, etc, so you cannot run bin/test.bash. So you first need to checkout languages before testing, but you also need to be on the current feature branch, which can be done through ``` github.head_ref ```, and now if you run the above code, the next issue is the naming of the container being "work". If you leave it as "work", the tests will not be able to cd properly from setup(), because the file must include "languages" and I have tried to remove that condition, but two tests, OBJ and V3 cd into a different location that includes their names, which is an issue. So I changed the name of "work" to "languages," and tested it again, and now the issue is that there seems to be an issue creating the Java directory from plcc, as the error message is ``` Java: error creating destination subdirectory ``` after each test tries to use plcc or plccmk. This is the part where I am stuck at now, I am not sure how to troubleshoot what exactly is going wrong as the code in plcc.py catches the error and outputs that default message. 
```
    try:
        os.mkdir(dst)
        debug('[lexFinishUp] ' + dst + ': destination subdirectory created')
    except FileExistsError:
        debug('[lexFinishUp] ' + dst + ': destination subdirectory exists')
    except:
        death(dst + ': error creating destination subdirectory')
``` 
Any thoughts? @StoneyJackson